plugins/database/mongodb/connection_producer.go

```diff
@@ -153,7 +158,7 @@ func (c *mongoDBConnectionProducer) Close() error {
 	return nil
 }
 
-func parseMongoURL(rawURL string) (*mgo.DialInfo, error) {
+func parseMongoURL(rawURL string, sslCert string, sslKey string, sslCA string) (*mgo.DialInfo, error) {
```

Comment:
If all variable are the same type, you can just repeat the variable names and put the type at the end.

Reply:
Great advice, thanks